<ui version="4.0" >
 <class>AghPhoneUI</class>
 <widget class="QWidget" name="AghPhoneUI" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>279</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>AghPhone</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>IP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ip_part_1" />
     </item>
     <item>
      <widget class="QLineEdit" name="ip_part_2" />
     </item>
     <item>
      <widget class="QLineEdit" name="ip_part_3" />
     </item>
     <item>
      <widget class="QLineEdit" name="ip_part_4" />
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Local port</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="local_port" />
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Remote port</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="remote_port" />
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="callButton" >
       <property name="text" >
        <string>Call</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="endCallButton" >
       <property name="text" >
        <string>End Call</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
